plugins/setting-assets/lang/en.json

```diff
@@ -25,6 +25,9 @@
     "Settings": "Settings",
     "SelectWorkspace": "Select workspace",
     "DeleteStatus": "Delete status",
-    "DeleteStatusConfirm": "Do you want to delete this status?"
+    "DeleteStatusConfirm": "Do you want to delete this status?",
+    "Reconnect": "Reconnect",
+    "IntegrationDisabled": " has been disabled",
```

Comment:
Why not use {substitution} for word translation?

Reply:
Because integration label is IntlString, and we must get and translate it before